.github/workflows/mpi-operator-docker-image-publish.yml

```diff
@@ -31,6 +31,7 @@ jobs:
             type=ref,event=pr
             type=semver,pattern={{version}}
             type=semver,pattern={{major}}.{{minor}}
+            type=raw,latest
```

Comment:
but wouldn't this affect every tag?

Reply:
Does that mean the existing `latest` tag image is overwritten?

Reply:
IIUC, by doing this the latest tag would always be overriden with every push. I think we should do this manually for new releases only.

Reply:
Makes sense. What do you think about replacing the image tag with `master` in the following to avoid [this](https://github.com/kubeflow/mpi-operator/issues/508) confusing?

https://github.com/kubeflow/mpi-operator/blob/05ac6addc0830e34c01a6cb13c25e25ac7f81fe6/manifests/overlays/standalone/kustomization.yaml#L15

Reply:
that sounds better to me.

The manifests that are part of a release should have the specific release tag.

Reply:
Sounds good. I would update this PR.